### DISCUSSION TITLE
[Support]: Motion threshold and contour area sliders do not match bounds in Tuning Motion Detection

### DISCUSSION THREAD

**Discussion by USER19464727** [blakeblackshear/frigate#15126] [General Support] [answered]
### Checklist

- [X] I have updated to the latest available Frigate version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [X] I have tried reproducing the issue in [incognito mode](https://www.computerworld.com/article/1719851/how-to-go-incognito-in-chrome-firefox-safari-and-edge.html) to rule out problems with any third party extensions or plugins I have installed.

### Describe the problem you are having

The slider for motion threshold for a camera goes from 5 to 80 and contour area from 5 to 100. These values do not match what is stated in the comments of the Tuning Motion Detection guide. If this is unintentional, I suggest having the slider match the available bounds for the variable.
https://docs.frigate.video/configuration/motion_detection/

### Steps to reproduce

1. Go to motion tuner for a camera.
2. Utilize sliders

### Version

0.14.1

### In which browser(s) are you experiencing the issue with?

HASS

### Frigate config file

```yaml
N/A
```

### docker-compose file or Docker CLI command

```yaml
N/A
```

### Relevant Frigate log output

```shell
N/A
```

### Relevant go2rtc log output

```shell
N/A
```

### Operating system

HassOS

### Install method

HassOS Addon

### Network connection

Wired

### Camera make and model

N/A

### Screenshots of the Frigate UI's System metrics pages

N/A

### Any other information that may be helpful

N/A

**Comment by hawkeye217** [comment, marked as answer]
This is intentional so as to make it easier to move the slider to more common values for most users. If the range was larger in the UI, it would be a more frustrating experience (especially for mobile users) to fine tune with the slider. 

If you truly need larger values, adjust via MQTT or the config file directly.
